app/components/post_textbox/post_textbox.js

```diff
@@ -98,6 +142,20 @@ export default class PostTextbox extends React.PureComponent {
         return (
             <View style={{padding: 7}}>
                 <Autocomplete ref={this.attachAutocomplete}/>
+                <View>
```

Comment:
Looks like the ```<Autocomplete/>``` component needs to be placed after the typing component.

![image](https://cloud.githubusercontent.com/assets/5090577/23136784/eb42ca50-f763-11e6-9bc1-8e4a995dc238.png)